### PR TITLE
[tests-only] added spaces examples for features inside `apiWebdavLocksUnlock` suite

### PR DESCRIPTION
**Pull request by kiranparajuli589**
## Description
Added spaces examples for features inside `apiWebdavLocksUnlock` suite

## Related Issue
- https://github.com/owncloud/ocis/issues/3098

## How Has This Been Tested?

- :robot: 

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Database schema changes (next release will require increase of minor version instead of patch)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [x] Tests only (no source changes)

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [x] Acceptance tests added
- [ ] Documentation ticket raised: <link> 
- [ ] Changelog item, see [TEMPLATE](https://github.com/owncloud/core/blob/master/changelog/TEMPLATE)

### CODE REVIEW

**Review by amrita-shrestha — APPROVED** [review]
LGTM

File: tests/acceptance/features/apiWebdavLocksUnlock/unlock.feature [hunk 1]
```diff
@@ -31,12 +39,20 @@ Feature: UNLOCK locked items
...
     Examples:
       | dav-path |
       | old      |
       | new      |
 
+    @personalSpace @skipOnOcV10
+    Examples:
+      | dav-path |
+      | spaces   |
+      | spaces   |
```
Comment (phil-davis) [inline tests/acceptance/features/apiWebdavLocksUnlock/unlock.feature:53]:
```suggestion
      | spaces   |
```

File: tests/acceptance/features/apiWebdavLocksUnlock/unlockSharingToShares.feature [hunk 2]
```diff
@@ -38,6 +38,12 @@ Feature: UNLOCK locked items (sharing)
       | new      | shared     | /PARENT/parent.txt |
       | new      | exclusive  | /PARENT/parent.txt |
 
+    @personalSpace @skipOnOcV10
+    Examples:
+      | dav-path | lock-scope | pending_share_path |
+      | spaces   | shared     | /PARENT/parent.txt |
+      | spaces   | exclusive  | /PARENT/parent.txt |
```
Comment (phil-davis) [inline tests/acceptance/features/apiWebdavLocksUnlock/unlockSharingToShares.feature:45]:
```suggestion
      | spaces   | shared     | /parent.txt |
      | spaces   | exclusive  | /parent.txt |
```

IMO we want to copy the "old" and "new" examples that are running on oC10.9.* and on oCIS.

Comment (kiranparajuli589) [reply tests/acceptance/features/apiWebdavLocksUnlock/unlockSharingToShares.feature]: fixed

File: tests/acceptance/features/apiWebdavLocksUnlock/unlock.feature (hunk 1 above)
Comment (kiranparajuli589) [reply tests/acceptance/features/apiWebdavLocksUnlock/unlock.feature]: fixed

**Comment by kiranparajuli589** [comment]
```console
+ wait-for-it -t 800 oc-server:8080
services aren't ready in 13m20s
```

https://drone.owncloud.com/owncloud/core/34595/34/10

build restarted

**Review by phil-davis — APPROVED** [review]

**Pull request merged**